pkg/services/object/get/get_test.go

```diff
@@ -963,7 +963,7 @@ func TestGetRemoteSmall(t *testing.T) {
 				addr.SetObject(oidtest.ID())
 
 				srcObj := generateObject(addr, nil, nil)
-				srcObj.SetPayloadSize(10)
+				srcObj.SetPayloadSize(11)
```

Comment:
What is the reason for this change?

Reply:
incorrect test: parent length is `10`, the first child obj's len is `10`, test expects another (the second) request to the child obj (it will not happen after the fix, but will happen if the parent obj is longer than the last child)